Tests/Recurly/Pager_Test.php

```diff
@@ -69,8 +69,6 @@ public function testFromHref() {
     $pager->_loadFrom($url);
 
     $this->assertEquals($url, $pager->getHref());
-    $this->assertEquals(6, $pager->count(), 'Returns correct count');
-    $this->assertEquals(6, count($pager), 'Returns correct count');
```

Comment:
I think we still want to keep this interface and we should just update the tests accordingly